iodata/overlap.py

```diff
@@ -156,6 +214,17 @@ def gob_cart_normalization(alpha: np.ndarray, n: np.ndarray) -> np.ndarray:
         The normalization constant for the gaussian cartesian basis.
 
     """
-    vfac2 = np.vectorize(factorialk)
-    return np.sqrt((4 * alpha)**sum(n) * (2 * alpha / np.pi)**1.5
-                   / np.prod(vfac2(2 * n - 1, 2)))
+    vfac2 = np.vectorize(factorial2)
```

Comment:
I'm not sure if this is needed. It may be a leftover from the old implementation. The `factorial2` function is already vectorized, e.g.

```python
>>> from scipy.special import factorial2
>>> factorial2([3, 5])
array([ 3., 15.])

```

Reply:
I can change that and time it to make sure, but, I believe you are correct. I tried replacing `np.vectorize` with `np.frompyfunc` since the latter makes things faster and the former makes them easier but is no faster than loops. It did not change the speed - therefore, I believe you are correct and neither was doing any additional vectorization.

Reply:
ok, makes sense.